exec-analysis/src/main/java/com/capitalone/dashboard/exec/collector/DefaultMetricCollector.java

```diff
@@ -102,15 +103,15 @@ public void collect(SparkSession sparkSession, JavaSparkContext javaSparkContext
 
         List<String> collectorItemList = new ArrayList<>();
         Optional.ofNullable(portfolioList).orElseGet(Collections::emptyList).stream()
-        .map(Portfolio::getProducts)
+                .map(Portfolio::getProducts)
```

Comment:
Can you revert the styling changes here?

Reply:
Reverted the styling to original state